src/components/Button.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button :class="['button', {'button--icon': icon,'button--left': align === 'left', 'button--right': align === 'right'} ]">
+  <button :disabled="disabled" :class="['button', {'button--icon': icon,'button--left': align === 'left', 'button--right': align === 'right'} ]">
```

Comment:
```suggestion
  <button
    :disabled="disabled"
    :class="[
      'button',
      {
        'button--icon': icon,
        'button--left': align === 'left',
        'button--right': align === 'right',
      },
    ]"
  >
```

This is quite a chunky line now, it might be worth splitting it out like this to make it a bit more readable